mobile/src/main/res/values-de/strings.xml

```diff
@@ -1,139 +1,138 @@
+  <string name="mainmenu_openhab_selectsitemap">Standard-Sitemap auswählen</string>
+  <string name="mainmenu_openhab_writetag">NFC-Tag beschreiben</string>
+  <string name="mainmenu_openhab_clearcache">Bilder-Cache leeren</string>
+  <!-- App settings strings -->
+  <string name="settings_connection_title">Verbindungseinstellungen</string>
+  <string name="settings_display_title">Display-Einstellungen</string>
+  <string name="settings_misc_title">Sonstige Einstellungen</string>
+  <string name="settings_openhab_url">openHAB URL</string>
+  <string name="settings_openhab_url_summary">URL des openHAB-Servers. Wenn gesetzt, ist Auto-Discovery deaktiviert. [%s]</string>
+  <string name="settings_openhab_alturl">openHAB Remote-URL</string>
+  <string name="settings_openhab_alturl_summary">URL eines openHAB-Servers außerhalb des lokalen Netzwerkes. Wird genutzt, wenn im lokalen Netzwerk kein openHAB-Server gefunden wurde. [%s]</string>
+  <string name="settings_openhab_username">Benutzername</string>
+  <string name="settings_openhab_password">Passwort</string>
+  <string name="settings_openhab_sslclientcert">SSL-Client-Zertifikat</string>
+  <string name="settings_openhab_sslclientcert_howto">Erläuterungen zum Client-Zertifikat</string>
+  <string name="settings_openhab_sslclientcert_howto_summary">Für Hinweise und Erläuterungen zur SSL-Authentifizierung mittels Client-Zertifikat hier tippen</string>
+  <string name="settings_openhab_screentimeroff">Ruhezustand deaktivieren</string>
+  <string name="settings_openhab_screentimeroff_summary">Verhindere den automatischen Ruhezustand bei Inaktivität wenn die openHAB App aktiv ist</string>
+  <string name="settings_openhab_demomode">Demomodus</string>
+  <string name="settings_openhab_demomode_summary">Demo-Daten an Stelle von realen Server-Daten verwenden</string>
+  <string name="settings_openhab_theme">Design</string>
+  <string name="settings_openhab_sslcert">SSL-Zertifikat ignorieren</string>
+  <string name="settings_openhab_sslcert_summary">Beim SSL-Handshake alle Zertifikate unabhängig von ihrer Gültigkeit akzeptieren</string>
+  <string name="settings_openhab_sslhost">SSL-Hostnamen ignorieren</string>
+  <string name="settings_openhab_sslhost_summary">Während des SSL-Handshakes keine Überprüfung des Hostnamens durchführen</string>
+  <string name="settings_openhab_fullscreen">Vollbild</string>
+  <string name="settings_openhab_fullscreen_summary">Geräte-Statusleiste während der Verwendung der App ausblenden</string>
+  <string name="settings_openhab_icon_format">Symbol-Format</string>
+  <string name="settings_openhab_icon_format_png">Pixelgrafik</string>
+  <string name="settings_openhab_icon_format_svg">Vektorgrafik</string>
+  <string name="settings_ringtone">Klingelton</string>
+  <string name="settings_openhab_sslsettings">Verschlüsselungs-Einstellungen</string>
+  <string name="settings_openhab_alt_connection">Globale Verbindungsdaten</string>
```

Comment:
Hmm, the german "global" has a different meaning than the english "remote", which I would translate with "Remote", only. Adding to crowdin :P

Reply:
Yeah you have a point. Sorry for that. Feel free to "disapprove" 😉

Reply:
One more thought: "Remote" is not a German word, nor is it a common english word used in German. How about "Fernzugriff" resp. "... aus der Ferne"

Reply:
Puh, I wouldn't use any of the respective german words either (that's why I used remote), the german words sounds pretty ugly in that context :(